upgrade/rendering-changes/6-3-0.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 6.3.0
+description: Handle the rendering changes in the 6.3.0 release of the Telerik UI for Blazor components.
+page_title: Rendering Changes in the 6.3.0 Release
+slug: changes-in-6-3-0
+position: 970
+---
+
+# Rendering Changes in 6.3.0
```

Comment:
`foo` class, but `.foo` element or even `div.foo` element. Consider using correct CSS selectors together with the HTML tag name.

Reply:
Updated in https://github.com/telerik/blazor-docs/pull/2429/commits/1cb1ec03338c6466d2929a4e7898a7e03916b5bb